config/README.md

```diff
@@ -1,102 +1,103 @@
+    CockroachDB nodes in the cluster.
+
+Note: we are investigating the use of service mesh frameworks to alleviate some
+of this operational overhead.
 
 ## Prerequisites
 
-Download & install:
+Download & install the following tools to your workstation:
 
-*   helm
+*   The [helm client](https://helm.sh/docs/using_helm/#installing-the-helm-client)
+    *   Required if deploying using the defined Kubernetes templates.
 *   kubectl
+    *   Required if deploying with Kubernetes.
 *   docker
-*   cockroachdb
+    *   Required if building new images of the DSS.
+*   Cockroachdb
+    *   Required to generate new CockroachDB certificates.
 *   Google Cloud SDK (if deploying on GCP)
-*   [Optional] Golang. Recommended to understand go, and the go toolchain.
+    * Required if deploying to Google Cloud.
+*   Golang.
+    *   Required if developing the DSS codebase.
 
 
 ## Building Docker images
 
-The grpc-backend and http-gateway binaries are built as docker images and pushed
-to a docker registry.  Which docker registry you use will depend on your
-individual requirements and choice of cloud platform.
 
-### Google Container Registry
+The grpc-backend and http-gateway are the 2 main binaries for processing DSS 
+requests. These are both built and pushed to a docker registry of your choice.
+You can easily find out how to push to a docker registry through a quick search.
+All major cloud providers have a docker registry service, or you can set up your
+own.
 
-List existing images:
+TODO(): The linux foundation should provide a default docker registry, with
+golden build versions.
 
-    gcloud --project $CLOUD_PROJECT container images list
-
-List the tags on an existing image:
-
-    gcloud --project $CLOUD_PROJECT container images list-tags gcr.io/$CLOUD_PROJECT/http-gateway
-
-Build a new image:
-
-    docker build -f cmds/http-gateway/Dockerfile  . -t gcr.io/$CLOUD_PROJECT/http-gateway:$VERSION
-    docker build -f cmds/grpc-backend/Dockerfile  . -t gcr.io/$CLOUD_PROJECT/grpc-backend:$VERSION
-
-Push your new image to Google Container Registry:
-
-    docker push gcr.io/$CLOUD_PROJECT/http-gateway:$VERSION
-    docker push gcr.io/$CLOUD_PROJECT/grpc-backend:$VERSION
+Set the environment variable `DOCKER_URL` to your docker registry url endpoint.
 
 Use the `build.sh` script in this directory to build and push an image tagged
 with the current date and git commit hash.
 
 
 ## Creating a new Kubernetes cluster
 
-You need to create a Kubernetes cluster to run the cockroachdb instance and the
-gRPC and HTTP servers.  You can do this on any supported
-[cloud provider](https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/)
-or even on your own infrastructure.  Instructions for GCP are given below,
-consult the Kubernetes documentation for other providers.
+This secion discusses deploying a kubernetes service, although you can deploy 
+a DSS node however you like as long as it meets the above requirements. You can
```

Comment:
Let's mention exactly which requirements.  If the "CRDB requirements" section comment is adopted above, we can say "...meets the CRDB requirements above."

Reply:
done